### PR TITLE
Change branding and colors

### PR DESCRIPTION
**Pull request by ItsRealmy**
I changed the logo, changed the colors and changed the colors of the SVG's on the homepage.

### CODE REVIEW

**Comment by ChrisChrome** [comment]
I'm not sure if it has to be done in that repo but the community instances page still uses the old logo

**Review by ChrisChrome — APPROVED** [review]
lgtm

File: assets/css/style.css [hunk 1]
```diff
@@ -1,8 +1,9 @@
 @charset "UTF-8";
 
 :root {
-  --fosscord: #2e4eef;
-  --fosscord-hover: #2e4eefa9;
+  --fosscord: #FF5F00;
```
Comment (erkinalp) [inline assets/css/style.css:4]:
```suggestion
  --fosscord: #ff5f00;
```

**Review by erkinalp — CHANGES_REQUESTED** [review]
consistency in cases

**Review by erkinalp — APPROVED** [review]

**Review by Thesourtimes — APPROVED** [review]

**Review by TheArcaneBrony — APPROVED** [review]
Looks fine to me at first glance

**Pull request merged**